plugin/vim-erlang-tags.vim

```diff
@@ -42,6 +43,17 @@ function! s:GetExecuteCmd()
     return s:exec_script . script_opts
 endfunction
 
+function! GetOtpTagsPath()
+    let otppath = system(s:otppath_cmd)
+    let otptags = otppath . "/otptags"
+    return otptags
+endfunction
+
+function! AsyncBuildOtpTags()
+    let cmd = '(OTPPATH=`' . s:otppath_cmd . '` && ' . s:exec_script . ' -i $OTPPATH -o $OTPPATH/otptags)'
```

Comment:
Vim cannot necessarily write "$OTPPATH/otptags". For me `erl -noinput -eval 'io:format("~ts", [code:lib_dir()]), init:stop().'` returns `/usr/lib/erlang/lib`, which causes `AsyncBuildOtpTags` to silently fail.

Three suggestions:

- Some error handling to report this situation.
- Maybe use the ":terminal" command instead of `system` + `&`? (It was added in Vim 8.1.)
- Maybe make it configurable to put the tags file somewhere else.